pkg/gadgets/seccomp/tracer/tracer.go

```diff
@@ -81,6 +82,7 @@ func (t *Tracer) Peek(mntns uint64) []byte {
 	// we need to test b==nil too
 	if b == nil {
 		// The container just hasn't done any syscall
+		log.Errorf("No syscall found with mntns %d", mntns)
```

Comment:
This seems unrelated, but it is harmless :)

Reply:
This error case happened when `progLink` was closed by the garbage collector, so the error message will help to detect this if this bug happens again.

Reply:
Ohh, cool!